install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -168,56 +321,89 @@ To enable writing to NFS volumes with SELinux enforcing on each node, run:
 # setsebool -P virt_use_nfs 1
```

Comment:
@pmorie I did *not* need to set `virt_sandbox_use_nfs on` in my testing. Are you certain? I'm wondering if there's something else going on in my test env...